src/tree_schema_free.c

```diff
@@ -513,6 +513,10 @@ lysc_extension_free(struct ly_ctx *ctx, struct lysc_ext **ext)
 void
 lysc_ext_instance_free(struct ly_ctx *ctx, struct lysc_ext_instance *ext)
 {
+    if (ly_ctx_get_options(ctx) & LY_CTX_SET_PRIV_PARSED) {
```

Comment:
Useless to call an API function internally and it is better to access the flags directly. In this case it does not matter because the function does not do anything but in general there may be some additional code that you usually do not want to execute internally (such as in `ly_ctx_set_options()`).